### PR TITLE
Ng9.1

### PR DESCRIPTION
**Pull request by Hypercubed**

### CODE REVIEW

File: projects/swimlane/ngx-ui/src/lib/components/code-editor/code-editor.component.ts [hunk 1]
```diff
@@ -61,8 +61,8 @@ const CODEMIRROR_VALUE_ACCESSOR = {
     './foldgutter.css',
     './dracula.css',
     './hint.scss',
-    './code-editor.component.scss'
-  ]
+    './code-editor.component.scss',
+  ],
```
Comment (marjan-georgiev) [inline projects/swimlane/ngx-ui/src/lib/components/code-editor/code-editor.component.ts:65]: Did our prettier config change to include trailing commas?

Comment (Hypercubed) [reply projects/swimlane/ngx-ui/src/lib/components/code-editor/code-editor.component.ts:65]: Yes

Comment (Hypercubed) [reply projects/swimlane/ngx-ui/src/lib/components/code-editor/code-editor.component.ts:65]: Actually.. not a change in or config but in prettier itself.

Comment (Hypercubed) [reply projects/swimlane/ngx-ui/src/lib/components/code-editor/code-editor.component.ts:65]: https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker

File: projects/swimlane/ngx-ui/src/lib/components/nag/nag.component.spec.ts [hunk 2]
```diff
@@ -48,12 +48,16 @@ describe('NagComponent', () => {
     expect(component.stateChanged.emit).toHaveBeenCalledTimes(2);
   });
 
-  it('can peek into component if watch is set', () => {
+  // TODO: thius test is flaky
```
Comment (marjan-georgiev) [inline projects/swimlane/ngx-ui/src/lib/components/nag/nag.component.spec.ts:51]: Are these changes needed after the nag PR gets merged?

Comment (Hypercubed) [reply projects/swimlane/ngx-ui/src/lib/components/nag/nag.component.spec.ts:51]: no...
